tests/unittests/CoreText/CTTypesetterTests.mm

```diff
@@ -0,0 +1,81 @@
+
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import <UIKit/UIKit.h>
+#include <TestFramework.h>
+#import <CoreFoundation/CoreFoundation.h>
+
+static NSAttributedString* getMultilineAttributedString() {
+    UIFontDescriptor* fontDescriptor = [UIFontDescriptor fontDescriptorWithName:@"Times New Roman" size:40];
+    UIFont* font = [UIFont fontWithDescriptor:fontDescriptor size:40];
+
+    NSRange wholeRange = NSMakeRange(0, 20);
+    NSMutableAttributedString* string = [[NSMutableAttributedString alloc] initWithString:@"hello \nworld goodbye"];
```

Comment:
>[[NSMutableAttributedString alloc] initWithString:@"hello \nworld goodbye"]; [](start = 40, length = 76)

Should autorelease here so that this isn't leaked.